lib/duckdb/prepared_statement.rb

```diff
@@ -45,17 +45,16 @@ def bind_date(i, value)
 
     def bind_time(i, value)
       case value
-      when Time
+      when DateTime, Time
         time = value
       else
         begin
-          time = Time.parse(value)
+          time = DateTime.parse(value)
```

Comment:
Can we use `Time.parse` ?

`DateTime` is deprecated with latest Ruby.

https://bugs.ruby-lang.org/issues/15712#note-4
https://github.com/ruby/date/commit/58ca6e6a3ee20c72a77266e0f74920b12a06ee9d

Reply:
We may close this issue, but I believe that is better that we use `strftime()` because dividing `nsec` by 1000 isn't the Ruby way, plus it makes the code unreadable. I will remove the DateTime from the `bind_time` with a commit soon, that is for sure.